src/main/java/org/example/Graphics-2D/NepalFlag.java

```diff
@@ -0,0 +1,44 @@
+import javax.swing.JFrame;
```

Comment:
Hi @SagunDevkota, 

- Rename _Graphics-2D_ so that it contains only lower case such as _graphics2d_ to follow package naming convention.

- Include package info at top of the class as :
 ```java 
package org.example.graphics2d;
```

Thanks